Logging to shared log directory: /tmp/test_logs//1331863807
only running test 12
excepting tests: 15c
Starting client mft01: -o user_xattr,acl,flock mft01@tcp:/lustre /mnt/lustre2
Started clients mft01: 
mft01@tcp:/lustre on /mnt/lustre2 type lustre (rw,user_xattr,acl,flock)
debug=-1
subsystem_debug=0xffb7e3ff
debug_mb=2
mft01: Checking config lustre mounted on /mnt/lustre
Checking servers environments
Checking clients mft01 environments
Using TIMEOUT=20
disable quota as required
debug=-1
subsystem_debug=0xffb7e3ff
debug_mb=2
setting all flavor to null
already have total 8 null connections
............

== replay-dual test 12: open resend timeout == 04:10:21 (1331863821)
Filesystem           1K-blocks      Used Available Use% Mounted on
mft01@tcp:/lustre       374928     49528    305272  14% /mnt/lustre
multiop /mnt/lustre/f12 vmo_c
TMPPIPE=/tmp/multiop_open_wait_pipe.26832
fail_loc=0x80000302
Failing mds1 on node mft01
Stopping /mnt/mds1 (opts:) on mft01
affected facets: mds1
Failover mds1 to mft01
04:10:36 (1331863836) waiting for mft01 network 900 secs ...
04:10:36 (1331863836) network interface is UP
Starting mds1: -o loop,user_xattr,acl  /tmp/lustre-mdt1 /mnt/mds1
debug=-1
subsystem_debug=0xffb7e3ff
debug_mb=2
Started lustre-MDT0000
fail_loc=0
/mnt/lustre/f12
/mnt/lustre/f12 has type file OK
Resetting fail_loc on all nodes...done.
PASS 12 (20s)
...........== replay-dual replay-dual.sh test complete, duration 34 sec == 04:10:41 (1331863841)
Stopping clients: mft01 /mnt/lustre2 (opts:)
Stopping client mft01 /mnt/lustre2 opts:
